public/locales/en-US/translations.json

```diff
@@ -515,9 +515,9 @@
   },
   "statusbar": {
     "updateToLast": "Update to latest version",
-    "connected": "Connected",
-    "connecting": "Connecting",
-    "disconnected": "Disconnected"
+    "connected": "HF Connected",
+    "connecting": "HF Connecting",
+    "disconnected": "HF Disconnected"
```

Comment:
I think we already use statusbar.connected/disconnected keys. so should add new keys for HF statuses
` <p>{`WS ${(wsConnected && !wsConnInterrupted) ? t('statusbar.connected') : t('statusbar.disconnected')}`}</p>`